mantis-connectors/mantis-connector-iceberg/dependencies.lock

```diff
@@ -5,113 +5,14 @@
         }
     },
     "baseline-exact-dependencies-main": {
-        "com.netflix:mantis-rxnetty": {
```

Comment:
@sundargates are these from a past gradle dep changes? did we remove these deps from iceberg connector?